akkumulator/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/trenger/TrengerService.kt

```diff
@@ -0,0 +1,177 @@
+package no.nav.helsearbeidsgiver.inntektsmelding.trenger
```

Comment:
Jeg kunne ønsket meg mer tester på unit-nivå av denne servicen, mtp at det er ganske mye logikk her, og integrasjonstestene tar kun happy case (og er vanskelig og treigt å kjøre lokalt)

Reply:
Data kanal, event listener, og Failkanal er deket med unit tester. Alle løser er deket med unit tester. Effort / reward faktoren her er relativ lav. Ref "not so happy case scenrioene" jeg vil utvide integrasjonstesten med dem.